plugins/tiddlywiki/tour/tours/introduction-to-tiddlywiki/steps/welcome.tid

```diff
@@ -4,8 +4,8 @@ display-mode: fullscreen
 
 ! Hello!
 
-!! An interactive tour of ~TiddlyWiki
+!! An interactive tour of {{$:/config/Tours/IntroductionToTiddlyWiki/ProductName}}
 
-Welcome to our tour of ~TiddlyWiki.
+Welcome to our tour of {{$:/config/Tours/IntroductionToTiddlyWiki/ProductName}}.
 
-We hope you'll find ~TiddlyWiki a helpful and supportive tool. Let's get started!
+We hope you'll find {{$:/config/Tours/IntroductionToTiddlyWiki/ProductName}} a helpful and supportive tool. Let's get started!
```

Comment:
I'm not 100% sure, but would `{{$:/config/Tours/ProductName}}` not be enough?

Reply:
The product name is a configuration option that is specific to the "Introduction to TiddlyWiki", it is not a general configuration option of the tour plugin.